### DISCUSSION TITLE
Can't import code

### DISCUSSION THREAD

**Discussion by Huy-nguyenhoangquoc** [ente-io/ente#3700] [Q&A]
Can't import code (using both phone and PC). 
When choosing import plain text/encrypted they just didn't work (i can choose file but it just not work, the UI didn't even change). I have 2 account, 1 is the backup, now it impossible for me to import code for backup account (i just recently use this app, i never use it before so this is my 1st time import/export code).
Hope you guy fix it soon

**Comment by crKRoEYS** [comment]
Import from which App? Check this:
https://help.ente.io/auth/migration-guides/
